src/btree/bt_page.c

```diff
@@ -39,6 +39,15 @@ __wt_page_block_meta_init(WT_SESSION_IMPL *session, WT_PAGE_BLOCK_META *meta)
     WT_ASSERT(session, page_id >= WT_BLOCK_MIN_PAGE_ID);
 
     meta->page_id = page_id;
+    /* 0 means the page hasn't been reconciled. */
+    meta->reconciliation_id = 0;
+    /* TODO: hard code the checkpoint id to 1. */
+    meta->checkpoint_id = WT_DISAGG_METADATA_TABLE_ID;
```

Comment:
These are not table IDs, so using the constants here is confusing. Maybe just put 1 for checkpoint_id for now, and 0 for the others, as this is a new page without previous versions set?

Reply:
When a page is inited, base checkpoint id should be the same as checkpoint id?